src/Sulu/Bundle/ActivityBundle/Application/Subscriber/SetDomainEventUserSubscriber.php

```diff
@@ -18,15 +18,9 @@
 
 class SetDomainEventUserSubscriber implements EventSubscriberInterface
 {
-    /**
-     * @var Security|null
-     */
-    private $security;
-
     public function __construct(
-        ?Security $security
+        private Security|null $security
```

Comment:
Why I personally prefer this style CI currently like this more:

```suggestion
        private ?Security$security
```